app/src/main/java/com/sun/noteapp/ui/home/MainActivity.kt

```diff
@@ -106,12 +114,53 @@ class MainActivity : AppCompatActivity(),
         adapterStaggeredGrid.updateData(notes)
     }
 
+    override fun onNavigationItemSelected(item: MenuItem) = when (item.itemId) {
```

Comment:
Có từ 3 case thì mới nên dùng when
```suggestion
    override fun onNavigationItemSelected(item: MenuItem): Boolean {
        if (item.itemId == R.id.navItemTrash) {
            startActivity(TrashActivity.getIntent(this))
            return true
        }
        return false
```

Reply:
when này có nhiều case nhưng mà mình để pull sau đẩy nốt